word_ladder/cli.py

```diff
@@ -14,7 +14,7 @@ def cli():
     """Entrypoint for word-ladder cli."""
 
 
-@cli.command(help="calculate the shortest word ladder" "between two words")
+@cli.command(help="calculate the shortest word ladder " "between two words")
```

Comment:
can you delete the excessive quotes that resulted from black undoing my linebreak?

Reply:
sure thing, wasn't sure if black was going to do things to it